mackup/applications/rclone.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = rclone
+
+[configuration_files]
+.config/rclone
```

Comment:
This needs to be moved under `xdg_configuration_files`, see https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory

I'll do it.